lib/text.rb

```diff
@@ -0,0 +1,16 @@
+require 'rubygems'
+require 'twilio-ruby'
+require_relative '../../.env.rb'
+
+module Text
+
+  def send_text
+    @client = Twilio::REST::Client.new ENV[:account_sid], ENV[:auth_token]
+
+    message = @client.account.messages.create(:body => "Thank you! Your order was placed and will be delivered before 17:35",
```

Comment:
Metrics/LineLength: Line is too long. [125/80]<br>Style/HashSyntax: Use the new Ruby 1.9 hash syntax.